packages/common/src/services/explore/Explore.ts

```diff
@@ -156,57 +129,30 @@ export class Explore {
     }
   }
 
-  // TODO(C-2719)
-  async getTopFolloweeSaves(limit = 25) {
-    try {
-      const libs = await this.audiusBackendInstance.getAudiusLibs()
-      const tracks: UserTrack[] =
-        await libs.discoveryProvider.getTopFolloweeSaves('track', limit, true)
-      return tracks
-    } catch (e) {
-      console.error(e)
-      return []
-    }
-  }
-
-  // TODO(C-2719)
   async getMostLovedTracks(userId: ID, limit = 25) {
     try {
-      const encodedUserId = encodeHashId(userId)
-      const libs = await this.audiusBackendInstance.getAudiusLibs()
-      const tracks: APITrack[] =
-        await libs.discoveryProvider.getMostLovedTracks(
-          encodedUserId,
-          limit,
-          true
-        )
-      return tracks.map(responseAdapter.makeTrack).filter(removeNullable)
+      const sdk = await this.audiusSdk()
+      const { data = [] } = await sdk.full.tracks.getMostLovedTracks({
+        limit,
+        withUsers: true,
```

Comment:
Well this is weird, I thought we were doing away with this parameter on v1 endpoints and just using the presence of `userId` to indicate we wanted to return users. But okay carry on :-)

Reply:
ah ok wasn't aware of that